reaching-the-venues.md

```diff
@@ -12,13 +12,24 @@ This page is to help the attendees reach Chennai and the conference venues.
 ### Getting to PyCon India 2019
 
 <br>
+<div class="img-zoom">
+     <img src="./assets/images/reaching-venue19-distance.png" 
+     	  	title="The Distance map" width="700" height="350" />
+</div>
 
 #### **By Plane**
 <br>
 
 Chennai has an international airport that is well connected to Europe, the Middle East, US,
 and South East Asia. It is also well connected by domestic flights.
 
+<div class="img-zoom">
+     <img src="./assets/images/reaching-venue19-from-chennai-airport.png" 
```

Comment:
Assets might get revisioned.

Use the `{% asset %}` helper like here.
https://github.com/pythonindia/inpycon2019/blob/5e5a1709ab4384dbdac26067f8675114b4cb4d0f/index.md#L8

Reply:
Modified with template